.goreleaser.yml

```diff
@@ -19,7 +19,7 @@ builds:
       - -X github.com/ethersphere/bee/v2.commitHash={{ .ShortCommit }}
       - -X github.com/ethersphere/bee/v2.commitTime={{ .CommitTimestamp }}
       - -X github.com/ethersphere/bee/v2/pkg/api.Version={{ .Env.BEE_API_VERSION }}
-      - -X github.com/ethersphere/bee/v2/pkg/debugapi.Version={{ .Env.BEE_DEBUG_API_VERSION }}
```

Comment:
This is to fix point 5 from https://github.com/ethersphere/bee/issues/4617?

Reply:
yes